server/test/services/energy-monitoring/energy-monitoring.calculateCostEveryThirtyMinutes.test.js

```
@@ -1,11 +1,14 @@
+const sinon = require('sinon');
+
+const { fake, assert } = sinon;
+const { expect } = require('chai');
 const EventEmitter = require('events');
 const dayjs = require('dayjs');
 const utc = require('dayjs/plugin/utc');
 const timezone = require('dayjs/plugin/timezone');
 
-dayjs.extend(utc);
-dayjs.extend(timezone);
+const { fake, assert } = sinon;
```

Biome (2.4.4)

[error] Shouldn't redeclare 'fake'. Consider to delete it or rename it.

(lint/suspicious/noRedeclare)